dev/DebugView.php

```diff
@@ -300,6 +300,9 @@ public function renderSourceFragment($lines, $errline) {
 	 * @return string
 	 */
 	public function renderTrace($trace) {
+		if(empty($trace)) {
+			return '';
+		}
```

Comment:
The code below returns a `</div>`, so does the caller except the closing div? Is there a reason this is just (basically) a silent error swallow rather than returning "Trace: no trace object passed" or something?